src/v/model/namespace.h

```diff
@@ -80,6 +80,10 @@ inline const model::topic
 inline const model::topic_namespace transform_log_internal_nt(
   model::kafka_namespace, model::transform_log_internal_topic);
 
+inline const model::topic datalake_coordinator_topic("datalake_coordinator");
+inline const model::topic_namespace datalake_coordinator_nt(
+  model::redpanda_ns, model::datalake_coordinator_topic);
```

Comment:
nit: all the internal topics are in `kafka_internal_namespace` only controller and kvstore lives in redpanda

Reply:
i see Noah suggested that to be `redpanda`